chsdi/models/vector/bafu.py

```diff
@@ -27,6 +27,83 @@ class Hydrogeologischekarte100(Base, Vector):
 register('ch.bafu.hydrogeologische-karte_100', Hydrogeologischekarte100)
 
 
+class KarstFliesswege(Base, Vector):
+    __tablename__ = 'karst_fliesswege'
+    __table_args__ = ({'schema': 'hydrologie', 'autoload': False})
+    __template__ = 'templates/htmlpopup/karst-unterirdische_fliesswege.mako'
+    __bodId__ = 'ch.bafu.karst-unterirdische_fliesswege'
+    __label__ = 'objectid'
+    __queryable_attributes__ = ['ei_type', 'ei_hdyn']
+    id = Column('bgdi_id', Integer, primary_key=True)
+    objectid = Column('objectid', Integer)
+    ei_id = Column('ei_id', Text)
+    ei_hdyn = Column('ei_hdyn', Text)
+    ei_type = Column('ei_type', Unicode)
+    the_geom = Column(Geometry2D)
+
+register('ch.bafu.karst-unterirdische_fliesswege', KarstFliesswege)
+
+
+class KarstEinzugsgebieteinheiten(Base, Vector):
+    __tablename__ = 'karst_einzugsgebieteinheiten'
+    __table_args__ = ({'schema': 'hydrologie', 'autoload': False})
+    __template__ = 'templates/htmlpopup/karst-einzugsgebietseinheiten.mako'
+    __bodId__ = 'ch.bafu.karst-einzugsgebietseinheiten'
+    __label__ = 'objectid'
+    __queryable_attributes__ = ['ub_type', 'ub_flux']
+    id = Column('bgdi_id', Integer, primary_key=True)
+    objectid = Column('objectid', Integer)
```

Comment:
why not use objectid instead of bgdi id for a primary key? Especially if you put stable ids. Use the provided ids otherwise they won't be stable (A Serial number that will change if you update the stable)